### PR TITLE
Fix document_types is not defined for Document-createElement.html

### PR DESCRIPTION
**Pull request by johndai1984**

### CODE REVIEW

File: custom-elements/Document-createElement.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 <link rel="help" content="https://dom.spec.whatwg.org/#concept-create-element">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
-<script src="resources/custom-elements-helper.js"></script>
+<script src="./resources/custom-elements-helpers.js"></script>
```
Comment (TakayoshiKochi) [inline custom-elements/Document-createElement.html]: This is not a strong rule, but as "./" is not required here, you can omit it.

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/19224)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/284078800)

**Comment by johndai1984** [comment]
Fixed nit.

**Review by TakayoshiKochi — APPROVED** [review]
LGTM

**Pull request merged**